Tools/ardupilotwaf/boards.py

```diff
@@ -17,6 +17,9 @@ def __init__(cls, name, bases, dct):
         if cls.abstract:
```

Comment:
I'm just putting the comment here because GitHub's new interface doesn't allow to post a comment for the whole commit in the PR's context :-(

Anyways. I believe this is a followup of https://github.com/diydrones/ardupilot/pull/3722#commitcomment-16592855, right? Thanks!

Although, I think I wasn't completely clear in my comment and I missed your followup questions while reviewing your responses. Sorry :-P

So:
> Adding a class attribute is just writing toolchain = "toolchain" in the class definition right?

Yes, and that's what you've here :-)

> But how would I get that definition in the toolchain tool?

So, that's where I think wasn't clear enough. My idea was that, in `Board`'s`configure()` function, we set `env.TOOLCHAIN = self.toolchain` just before loading compiler_{c,cxx}. If we don't do this, all places that need to know which toolchain is in use need to import board, which is an unnecessary dependency IMO. And I think the less  tools depend on each other, the better.